### ISSUE TITLE
references within initializer lists are not mentioned in member function pages (Origin: bugzilla #335587)

### ISSUE THREAD

**Issue by doxygen** [doxygen/doxygen#2088]
status NEW severity *normal* in component *build* for *---*
Reported in version *1.4.x* on platform *Other*
Assigned to: Dimitri van Heesch

On 2006-03-22 22:30:27 +0000, John Calcote wrote:

    Please describe the problem:
    References to other documented functions are not mentioned in the
    "References"
    list of a constructor when the reference is contained in an initializer
    list.
    For instance, if I have a constructor that looks like this:
    
       MyClass(void) : m_somemember(AnotherDocumentedFunction()) {}
    
    Then AnotherDocumentedFunction is not listed in the References section of
    MyClass::MyClass(void).
    
    Steps to reproduce:
    1. Define a class in a C++ header file with an inline body.
    2. Initialize some class data member by calling a documented global
    function.
    3. Run doxygen and check the references section of the doc page for the
    class
    constructor.
    
    
    Actual results:
    The reference to the function called in the initializer list is missing from
    the
    references section of the constructor.
    
    Expected results:
    The function called in the initializer list would be in the references
    section.
    
    Does this happen every time?
    Yes
    
    Other information:

**Comment by albert-github** [comment]
Having the small example:
```
/// \file

/// the class
class MyClass
{
  public:
    /// the constructor
    MyClass(void) : m_somemember(AnotherDocumentedFunction()) {}

    /// the initializer fie
    int AnotherDocumentedFunction() { return 42;}

    /// member var
    int m_somemember = -1;
};
```
We see with the doxygen version 1.12.0 (and similar in 1.4.6):
![image](https://github.com/user-attachments/assets/f45aa696-58a2-40ba-a0a5-3ec07a676aa6)

and with the current doxygen master (1.13.0 (e3975ef2bbf13098e3423a2c78315d5f145f3d13)):
![image](https://github.com/user-attachments/assets/8e3a95b0-6330-40df-bc2a-0bd036501619)


This looks all OK now to me.

I think the problem is fixed but not released.

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.13.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
